### DISCUSSION TITLE
提升（单线程）大文件线性读取速度

### DISCUSSION THREAD

**Discussion by ghost** [juicedata/juicefs#4168] [General]
我用
``juicefs mount  --cache-partial-only --buffer-size=8192 --prefetch=0 --max-uploads=200  --max-deletes=100  --update-fstab -d --backup-meta 8h  -o allow_other <....> /juicefsmount`` 
挂载了一个s3上的juicefs，metadata存在本地部署的tikv实验环境（均启用了ssl和证书验证）

然后 ``pv /juicefsmount/1.bin > /dev/null`` 发现读取速度和jucefs自带的benchmark（p=1）显示速度相同（500MB/s）但不如直接用s5 cmd直接下载s3服务上的大文件（3200MB/s）。

当jucefs自带的benchmark p=4时，读取速度可达到2000MB/s，多线程读取速度是能吃满带宽的。

于是怀疑是juicefs的读写metadata和ssl overhead导致了瓶颈？但增大``--prefetch``并无效果。文档中有readahead，但找不到修改readahead相关参数的地方。

**Comment by ghost** [comment]
求大佬指点，还有哪些可以调整的参数呀？

**Comment by michaelmyc** [comment]
增加block size试试？

**Reply by ghost** [reply]
这个需要重新格式化么？还是说对新数据生效？
